### ISSUE TITLE
macOS wrong path linking

### ISSUE THREAD

**Issue by mmaenz** [FlaxEngine/FlaxEngine#693]
I have searched for this but didn't find an existing issue.

**Issue description:**
Path to FlaxEditor.dylib points to non existing user directory.

```
mmaenz@DevMac Release % pwd
/Users/mmaenz/Downloads/FlaxEditorMac/Binaries/Editor/Mac/Release
mmaenz@DevMac Release % ./FlaxEditor  
dyld[3632]: Library not loaded: /Users/runner/work/FlaxEngine/FlaxEngine/Binaries/Editor/Mac/Release/FlaxEditor.dylib
  Referenced from: /Users/mmaenz/Downloads/FlaxEditorMac/Binaries/Editor/Mac/Release/FlaxEditor
  Reason: tried: '/Users/runner/work/FlaxEngine/FlaxEngine/Binaries/Editor/Mac/Release/FlaxEditor.dylib' (no such file), '/usr/local/lib/FlaxEditor.dylib' (no such file), '/usr/lib/FlaxEditor.dylib' (no such file)
zsh: abort      ./FlaxEditor
``` 
  
Output of `otool -l ./FlaxEngine`
```
Load command 20
          cmd LC_LOAD_DYLIB
      cmdsize 112
         name /Users/runner/work/FlaxEngine/FlaxEngine/Binaries/Editor/Mac/Release/FlaxEditor.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 0.0.0
compatibility version 0.0.0
```
The rpath is not set correct.

**Steps to reproduce:**
Download macOS artefact editor and try to run in terminal.

**Minimal reproduction project:**
Not needed

**Flax version:**
Irrelevant, problem is caused by cd-process when building the binaries.

**Ways to circumvent**
`install_name_tool -change "/Users/runner/work/FlaxEngine/FlaxEngine/Binaries/Editor/Mac/Release/FlaxEditor.dylib"  "@loader_path/../Release/FlaxEditor.dylib" ./FlaxEditor` results in  
```
Load command 20
          cmd LC_LOAD_DYLIB
      cmdsize 72
         name @loader_path/../Release/FlaxEditor.dylib (offset 24)
   time stamp 2 Thu Jan  1 01:00:02 1970
      current version 0.0.0
compatibility version 0.0.0

```  
and the editor starts.  
  
BTW it is so great to see Flax coming to macOS! Love it...

**Comment by mafiesto4** [comment]
Thanks for reporting this. I don't know how this wasn't found before even tho we tested some builds from Github Actions. Fixed in 09c995d67b9e30ddd2a9ce80b4edda46ef6d5c61 + aed6f0403d5b3a36518e940844811827417217a2.
I started a new build. Let me know if it works fine now for you 📦 😄

**Comment by mmaenz** [comment]
Yes it works. 

It just gives a bunch of errors on not allowed apps/libraries that can be allowed in System Preferences which could be solved later.

But it ends in an all over red window. Will open new Issue for that.
